src/components/Home.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react'
+import { ArticleCard } from './ArticleCard'
+
+export const Home = () => {
+
+const [articles,setArticles]=useState([])
+
+    useEffect(() => {
+        fetch(`https://news-website-0p9e.onrender.com/api/articles`).then((response) => {
+        return response.json()
+        }).then((response) => {
```

Comment:
I'd encourage you to put this into a util reusable function, like I showed in that end of day seminar last week. That way you can then re-use this function over and over again elsewhere.

Also, it's entirely your choice but you may want to use `Axios` as it'll help you handle errors easier later on.